From 28b859157eedaa23d5102253d9ba6ec2e387cad0 Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Tue, 23 Apr 2013 16:24:31 -0700
Subject: Fix an issue with apkcerts generation for target files archive

The name entry for each apk is missing the ".apk" suffix.
Adds it back so the sign_target_files_apks script works agin.

Change-Id: If9f8afff753338f492befb4556bfe6d509a86fe7
Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 core/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index 00eb01c..684847a 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -274,7 +274,7 @@ $(APKCERTS_FILE):
 				'$(p) $(PACKAGES.$(p).CERTIFICATE) $(PACKAGES.$(p).PRIVATE_KEY)')) ; \
 		do \
 			read pkg_name pkg_cer pkg_key <<<$${xline}; \
-			echo name=\"$${pkg_name}\" certificate=\"$${pkg_cer}\" private_key=\"$${pkg_key}\" >>$@ ; \
+			echo name=\"$${pkg_name}.apk\" certificate=\"$${pkg_cer}\" private_key=\"$${pkg_key}\" >>$@ ; \
 		done
 	# In case $(PACKAGES) is empty.
 	$(hide) touch $@
-- 
1.7.4.1

